ml_metadata/metadata_store/mlmd_types_test.py

```diff
@@ -33,7 +33,7 @@ def _get_execution_type_name(
   return extensions[metadata_store_pb2.system_type_extension].type_name
 
 
-class MlmdTypesTest(absltest.TestCase):
+class MlmdTypesTest(unittest.TestCase):
```

Comment:
Hmm, but `absltest.TestCase` is a child of `unittest.TestCase`: https://github.com/abseil/abseil-py/blob/296c08b0137fc3576cafbb0b85aea5fc06c391c1/absl/testing/absltest.py#L562C1-L562C35. Was there any issue in particular that you discovered with keeping `absltest.TestCase`?

Reply:
I had the same question. Pytest is able to collect `absltest.TestCase`s, as was done in TFX.